lib/resources/toml.rb

```diff
@@ -0,0 +1,33 @@
+# encoding: utf-8
+# author: Nolan Davidson
+
+require 'toml'
+
+# Parses a TOML document
+# Usage
+# describe toml('default.toml') do
```

Comment:
We do not need the comments, since the example is inline as part of the `example`